### PR TITLE
Add key short for save the file

### PR DESCRIPTION
**Pull request by MKFMIKU**

### CODE REVIEW

File: src/script.js [hunk 1]
```diff
@@ -176,7 +176,7 @@
 			e.preventDefault();
 		});
 
-		saveHtmlBtn.addEventListener('click', function (e) {
+		function save_file() {
```
Comment (chinchang) [inline src/script.js]: This project is written with camelCase convention. So `saveFile` instead. Also now that this is a named function, would be better off outside the `init` function. Put it alongside `init` function.

File: src/script.js [hunk 2]
```diff
@@ -197,6 +197,17 @@
...
+		saveHtmlBtn.addEventListener('click', function (e) {
+			save_file();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+			if ((event.ctrlKey)&&(event.keyCode==83)){
+				event.returnValue=false;
+				save_file();
+			 }
 		});
```
Comment (chinchang) [inline src/script.js]: Some spacing fixes required:

```
window.addEventListener('keydown', function (event) {
 if ((event.ctrlKey) && (event.keyCode == 83)) {
  event.returnValue = false;
  save_file();
 }
});
```

File: src/script.js [hunk 3]
```diff
@@ -197,6 +197,17 @@
...
 			a.remove();
+		};
+
+		saveHtmlBtn.addEventListener('click', function (e) {
+			save_file();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+			if ((event.ctrlKey)&&(event.keyCode==83)){
+				event.returnValue=false;
```
Comment (chinchang) [inline src/script.js]: What does this line do exactly?

**Comment by chinchang** [comment]
fixes #9

File: src/script.js [hunk 4]
```diff
@@ -197,6 +197,17 @@
...
 			a.click();
 			a.remove();
+		};
+
+		saveHtmlBtn.addEventListener('click', function (e) {
+			save_file();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+			if ((event.ctrlKey)&&(event.keyCode==83)){
```
Comment (chinchang) [inline src/script.js]: Mention a comment that this is code for `Ctrl + S` as 83 might not be recognizable to every developer.

File: src/script.js (hunk 3 above)
Comment (MKFMIKU) [inline src/script.js]: I just find Chrome has use `event.preventDefault()` to cancel the default key down event

File: src/script.js [hunk 5]
```diff
@@ -235,14 +235,14 @@
...
+		var html = editur.cm.html.getValue();
+		var css = editur.cm.css.getValue();
+		var js = editur.cm.js.getValue();
 
-			var fileContent = '<html><head>\n<style>\n'
-				+ css + '\n</style>\n</head>\n<body>\n'
-				+ html + '\n<script>\n' + js + '\n</script>\n\n</body>\n</html>';
+		var fileContent = '<html><head>\n<style>\n'
+			+ css + '\n</style>\n</head>\n<body>\n'
+			+ html + '\n<script>\n' + js + '\n</script>\n\n</body>\n</html>';
```
Comment (MKFMIKU) [inline src/script.js]: My VSCode has done this for me...How long should I make this space?

Comment (chinchang) [inline src/script.js]: No spaces. Use tabs.

File: src/script.js [hunk 6]
```diff
@@ -247,6 +235,29 @@
...
+	function saveFile() {
+		var html = editur.cm.html.getValue();
+		var css = editur.cm.css.getValue();
+		var js = editur.cm.js.getValue();
+
+		var fileContent = '<html><head>\n<style>\n'
+			+ css + '\n</style>\n</head>\n<body>\n'
+			+ html + '\n<script>\n' + js + '\n</script>\n\n</body>\n</html>';
+
+			var d = new Date();
```
Comment (chinchang) [inline src/script.js]: extra indentation in following lines.

**Comment by MKFMIKU** [comment]
I think there is no problem

File: src/script.js [hunk 7]
```diff
@@ -177,26 +177,14 @@
...
-			a.download = fileName;
-			a.style.display = 'none';
-			document.body.appendChild(a);
-			a.click();
-			a.remove();
+			saveFile();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+		  if ((event.ctrlKey)&&(event.keyCode==83)){
```
Comment (chinchang) [inline src/script.js]: tested this on my mac and this isn't working because there is no `ctrl` key on mac. You should check for `event.metaKey` instead.

**Comment by chinchang** [comment]
Did some fixes regarding eslint and opened this PR as #13. Closing this.

**Pull request closed without merging**